benchmark_runner/grafana/perf/jsonnet/main.libsonnet

```diff
@@ -357,13 +357,13 @@ g.dashboard.new('PerfCI-Regression-Summary')
           + elasticsearch.withMetrics([
             elasticsearch.metrics.MetricAggregationWithSettings.Max.withField('ci_minutes_time')
             + elasticsearch.metrics.MetricAggregationWithSettings.Max.withId('1')
-            + elasticsearch.metrics.MetricAggregationWithSettings.Max.settings.withScript('Integer.parseInt(\"0\"+doc["kata_rpm_version.keyword"].value.replace(\".\",\"\").replace(\"r\",\"\").replace(\"c\",\"\").replace(\"f\",\"\").replace(\"-\",\"\").replace(\"e\",\"1\").replace(\"c\",\"\").replace(\"r\",\"\"))')
+            + elasticsearch.metrics.MetricAggregationWithSettings.Max.settings.withScript('(doc["kata_rpm_version.keyword"].value.indexOf(\" \") == -1) ? Integer.parseInt(\"0\"+doc["kata_rpm_version.keyword"].value.replace(\".\",\"\").replace(\"r\",\"\").replace(\"c\",\"\").replace(\"f\",\"\").replace(\"-\",\"\")) : 0')
```

Comment:
This really needs a unit test with as wide a variety as possible of versions and expected mappings.  Otherwise it's going to break very easily.

And I really think you need to find a way to store this as something other than an integer, since the name is intrinsically structured (how do you do it for the OCP version?).  I'm not going to hold this for that, though.

Reply:
Thats a grafonnet code, no unittest for it.

Reply:
We can test the parsing of the values, though.  It's just python code.

Reply:
That should be in separate task, not part of this PR

Reply:
I disagree.  This is exactly the right point to add this unit test.  You've modified the parser, so there should be a test to ensure that the parser does the right thing.

Reply:
@RobertKrawitz, I disagree. 
I will not add unittest for 1 line.
Its a seperate project !!!

Reply:
I've had plenty of one liners that are so complex that a unit test is needed.  But OK, it isn't Python code, so it can be a separate issue.

Reply:
Yes, main.libsonnet is pure grafonnet code based on jsonnet.